includes/download-functions.php

```diff
@@ -789,7 +789,7 @@ function edd_record_download_in_log( $download_id = 0, $file_id = 0, $user_info
 
 	$user_agent = $browser->getBrowser() . ' ' . $browser->getVersion() . '/' . $browser->getPlatform();
 
-	edd_add_file_download_log( array(
+	$download_log = edd_add_file_download_log( array(
```

Comment:
```suggestion
	edd_add_file_download_log( array(
```

The log ID isn't needed.